files/en-us/web/css/css_functions/index.md

```diff
@@ -333,16 +333,14 @@ The following functions are used as a value of different `animation-timeline` pr
 
 ## Anchor positioning functions
 
-The anchor positioning functions are used when positioning and sizing **anchor-positioned elements** relative to the location and size of their associated **anchor elements**.
+The anchor positioning functions are used when positioning and sizing anchor-positioned elements relative to the location and size of their associated anchor elements.
 
 - {{cssxref("anchor", "anchor()")}}
   - : Returns a length relative to the position of the edges of an anchor-positioned element's associated anchor element.
 - {{cssxref("anchor-size", "anchor-size()")}}
-  - : Returns a length relative to the size dimensions of an anchor-positioned element's associated anchor element.
+  - : Returns a length relative to the size of the associated anchor element.
 - {{cssxref("inset-area_function", "inset-area()")}}
-  - : Returns a custom position try option based on the {{cssxref("inset-area")}} property value passed as a parameter.
-
-Each linked page details the function's syntax. For an introduction to [CSS anchor positioning](/en-US/docs/Web/CSS/CSS_anchor_positioning), see [Using CSS anchor positioning](/en-US/docs/Web/CSS/CSS_anchor_positioning/Using) and [Handling overflow: try options and conditional hiding](/en-US/docs/Web/CSS/CSS_anchor_positioning/Try_options_hiding) guides.
+  - : Defines the anchor element side of a position try option.
```

Comment:
The suggested updates to this page are based on following the structure of all the other function groupings on the page.